aries-site/src/pages/components/avatar.mdx

```diff
@@ -0,0 +1,111 @@
+import { BestPracticeGroup, Example } from '../../layouts';
+import {
+  AvatarAccessibilityExample,
+  AvatarCardExampleDo,
+  AvatarCardExampleDont,
+  AvatarDataTableExampleDo,
+  AvatarDataTableExampleDont,
+  AvatarPageHeaderExample,
+  AvatarSizePreviewExample,
+} from '../../examples';
+
+## Anatomy
+
+Avatars can be displayed as an image, initials, or a logo. They come in three different sizes:
+
+- `small`: 24x24px
```

Comment:
Can we do a table like on the content layout page? https://deploy-preview-2982--keen-mayer-a86c8b.netlify.app/templates/content-layouts?q=breakpoint#breakpoints

column titles would be "size" and "Dimensions (in pixels)"

Reply:
done